docs/source/user-guide/deployment/quick-start.md

```diff
@@ -0,0 +1,144 @@
+<!---
+  Licensed to the Apache Software Foundation (ASF) under one
+  or more contributor license agreements.  See the NOTICE file
+  distributed with this work for additional information
+  regarding copyright ownership.  The ASF licenses this file
+  to you under the Apache License, Version 2.0 (the
+  "License"); you may not use this file except in compliance
+  with the License.  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing,
+  software distributed under the License is distributed on an
+  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  KIND, either express or implied.  See the License for the
+  specific language governing permissions and limitations
+  under the License.
+-->
+
+# Ballista Quickstart 
+
+A simple way to start a local cluster for testing purposes is to use cargo to build the project and then run the scheduler and executor binaries directly along with the Ballista UI.
+
+If you open this application in the provided [Dev Container](https://containers.dev/overview) the required dependencies will already be installed namely Rust, Node.js and Yarn. If you are not using the Dev Container, you will need to install these dependencies yourself.
```

Comment:
What is a dev container, and how would I open one? I took a quick look at the link but the answer wasn't immediately clear and this may confuse people.

Reply:
A Dev container serves as Microsoft's solution for maintaining consistent development environments. You can learn more about it here: [Microsoft Dev Containers](https://code.visualstudio.com/docs/devcontainers/containers). Additionally, it supports a range of coding IDEs, which you can find listed [here](https://containers.dev/supporting.html).

I primarily utilize it to set up development sandboxes. If you have both VSCode and Docker installed on your machine, opening a project in VSCode that contains a devcontainer.json file will trigger a prompt, allowing you to open the workspace within a Docker container.

I thought it might speed up the development process for Ballista as everyone will be able to develop in the same environment. But I can also see how it's a pretty new technology that isn't widely adopted yet.

Reply:
Yes, I think two separate PRs would be great. I think we need to add some additional content around dev containers for those (like me) who have not encountered them before.